### PR TITLE
Add dev workflow scripts for build and enable ci

### PR DESCRIPTION
**Pull request by sbomer**
This adds dev workflow scripts following the example of coreclr.
- init-tools.cmd/sh retrieves the buildtools
- run.cmd/sh calls init-tools and invokes the downloaded run.exe
- config.json includes a run command that can be used to build the repo
- build.cmd calls the run script to build the repo
- netci.groovy sets up push and pr triggers for the windows build

### CODE REVIEW

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
@mmitche Why does "test ci please" have no effect? Do I need to wait for the generator to run again?

**Comment by mmitche** [comment]
@sbomer dotnet-bot doesn't have permissions to update status on the PR.

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by sbomer** [comment]
@dotnet-bot test ci please

**Comment by sbomer** [comment]
@dotnet-bot test ci please

**Comment by sbomer** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import jobs.generation.Utilities;
+
+def project = GithubProject
+def branch = GithubBranchName
+
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+
+        def newJob = job(Utilities.getFullJobName(project, os.toLowerCase(), isPR))
```
Comment (mmitche) [inline netci.groovy]: { brackets missing

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by sbomer** [comment]
test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dontet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by sbomer** [comment]
@marek-safar I'd like to merge this to test the ci builds.

**Pull request merged**